apps/aetx/src/aetx_env.erl

```diff
@@ -244,7 +244,7 @@ tx_event(Name, #env{events = Events} = Env) ->
         none -> Env;
         {value, SignedTx} ->
             TxHash = aetx_sign:hash(SignedTx),
-            {Type, _} = aetx:specialize_type(aetx_sign:tx(SignedTx)),
+            {Type, _} = aetx:specialize_type(aetx_sign:innermost_tx(SignedTx)),
```

Comment:
This is made GA aware. For transactions wrappend in a meta authentication transaction, the old processing was emitting not channel's type but rather `ga_meta_tx` instead.

Reply:
Based on inspection of the code this shall make the `aesc_fsm_min_depth_watcher` work better for the `has_tx` scenario.